modules/graphviz/0.0.2/MODULE.bazel

```diff
@@ -0,0 +1,36 @@
+module(
+    name = "graphviz",
+    version = "0.0.2",
+)
+
+
+bazel_dep(name = "rules_foreign_cc", version = "0.15.1")
+bazel_dep(name = "rules_cc", version = "0.2.8")
+bazel_dep(name = "zlib_foreign", version = "0.0.1", repo_name = "zlib")
+bazel_dep(name = "rules_shell", version = "0.6.1")
+
+
+bazel_dep(name = "hermetic_cc_toolchain", version = "3.1.0")
```

Comment:
This should be a dev_dependency, together with the `register_toolchain` call below.